internal/cli/client/client.go

```diff
@@ -2,10 +2,8 @@ package client
 
 import (
 	"context"
-
-	"errors"
 	"io"
-
+	octaviusErrors "octavius/internal/pkg/octavius_errors"
```

Comment:
there's no need to give the package name `ocatavios_` prefix. better rename it to `errors` only since we already use the import path.

Reply:
We named it as octavius_errors to differentiate it from the Golang errors library and remove any scope of confusion later on.

Reply:
We are thinking to rename it to `Errors`. is it ok ?

Reply:
there's will be no confusion, because you only use one in a time for example:

```go

// go's error import will be like this
import (
  "errors"
)
```

```go

// octavius's error import will be like this
import (
  "octavius/internal/pkg/errors"
)
```

people can read the import path, there will be no confusion.

it's the rare case both will be imported, and if so we can do aliasing

```go

import (
  "errors"
  octerr "octavius/internal/pkg/errors"
)


```

Reply:
@adbusa67 , yes please rename it to `errors` so the import path will be: `octavius/internal/pkg/errors`

Reply:
reference: https://golang.org/doc/effective_go.html#package-names

Reply:
We renamed octaviusErrors to errors and used octerr to avoid confusion while using.